src/language/__tests__/parser-test.js

```diff
@@ -391,6 +391,16 @@ describe('Parser', () => {
     expect(result.loc).to.equal(undefined);
   });
 
+  it('Experimental: allows parsing fragment defined variables', () => {
```

Comment:
Is it worth adding a test that verifies that without the `experimentalFragmentVariables: true` flag, this does **not** parse? Makes sure that we've gated the experimental parsing correctly. (applies to each of the tests, I suspect)

Reply:
This test case already has that (see my comment below).